CIP-0026/CIP-0026.md

```diff
@@ -0,0 +1,468 @@
+
+Note that:
+
+- In this scenario, a single valid signature is sufficient to consider a metadata entry valid but there can be many signatures (invalid or valid). So long as one is valid, the entry is considered verified.
+
+- The verification is done per _entry_. That is, a metadata object may contain both verified and unverified entries. Plus, entries under the same subject may be verified by different keys. 
+
+The way by which the trusted keys are registered into clients is unspecified although we already consider the following, **non-overlapping**, **complementary**, options:
+
+1. Clients MAY explicitly prompt the consumer / end-user about whether to accept a certain entry. While this is unpractical for some cases (e.g. token metadata), it may be relevant for some.
+
+1. Clients MAY come with a set of pre-configured well-known trusted keys chosen at the discretion of the application editor. 
+
+2. End-users SHOULD have the ability to add/remove keys from their trusted set. This allows end-users to introduce trusted keys they know before they end up in the pre-configured set (which likely follow the application release cycle). In this context, keys are advertised by the signing authority by some means, for instance, on social media or on another form of public key registry (e.g. [keybase.io](http://keybase.io/))
+
+3. Mappings of subjects to keys MAY be recorded on-chain, using transaction metadata and an appropriate label registered on [CIP-0010](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0010/CIP-0010.md). In some scenarios, the context within which the transaction is signed may be enough to reliably trust the legitimacy of a mapping. For example, in the case of a monetary policy, one could imagine _registering trusted keys_ in the same transaction minting tokens. Because the transaction is inserted in the ledger, it must have been signed by the token issuer and therefore, the specified keys are without doubt acknowledged by the token issuer. As a result, clients having access to on-chain data can automatically discover new mappings from observing the chain. 
+
+### Special Case: Phase-1 Monetary Policies
+
+> <sub>░ note ░</sub> 
+> 
+> This approach is now considered superseded by the more general approach. While it has some nice properties, new applications should consider sticking to the general case **even for phase-1 monetary policies**. Servers who seek backward-compatibility should implement both. 
+
+We consider the case of phase-1 monetary policies introduced during the Allegra era of Cardano. Such policies are identified by a simple (native) script which is validated by the ledger during the so-called phase-1 validations. Such scripts are made of key hashes, combined via a set of basic first-order logic primitives (ALL, ANY-OF, N-OF...) such that, it is possible to statically verify whether a set of signatures would validate the script. 
```

Comment:
I thought "native" was to avoid nowadays because everything is so-to-speak native, including Plutus :grimacing: ... Actually, the current sentence is even wrong because phase-1 monetary scripts were introduced in Shelley, Allegra only added the timelock extension. 

For lack of better  terminology for now, I'll stick to phase-1 monetary policies :sweat_smile: